### PR TITLE
sample: anthos baremetal user cluster terraform resource - manuallb

### PR DESCRIPTION
**Pull request by Shabirmean**
### Description
- This PR adds a new sample for the Anthos bare metal user-cluster terraform resource for creating clusters in **Manual LB** mode

### Note:
- Linting error is related to the provider being not public yet

### Related PR
- #324

### CODE REVIEW

File: gkeonprem_baremetal_cluster_manuallb/main.tf [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#[START gkeonprem_bare_metal_cluster_manuallb]
+# Fill in the local variables according to your baremetal environment; you must
```
Comment (msampathkumar) [inline gkeonprem_baremetal_cluster_manuallb/main.tf:18]: Can you update the sample without the use of `locals.`?

You can refer to other TF samples for reference

Comment (betsy-lichtenberg) [reply gkeonprem_baremetal_cluster_manuallb/main.tf:18]: Yes, please see guidance at [go/terraform-docs-samples-repo-style](http://goto.google.com/terraform-docs-samples-repo-style).

Comment (Shabirmean) [reply gkeonprem_baremetal_cluster_manuallb/main.tf:18]: Can I get access to that doc? I cannot view it

Comment (betsy-lichtenberg) [reply gkeonprem_baremetal_cluster_manuallb/main.tf:18]: The content has been ported to the following locations:

* https://googlecloudplatform.github.io/samples-style-guide/ 
* https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#review-terraform-sample-requirements-best-practices-and-code-style

**Comment by Shabirmean** [comment]
Closing this in favor of https://github.com/terraform-google-modules/terraform-docs-samples/pull/397 which is branched off of #324

**Pull request closed without merging**